stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/input/MessageInputView.kt

```diff
@@ -537,4 +570,24 @@ public class MessageInputView : ConstraintLayout {
     public interface BigFileSelectionListener {
         public fun handleBigFileSelected(hasBigFile: Boolean)
     }
+
+    /**
+     * Users lookup functional interface. Used to create custom users lookup algorithm.
+     */
+    public interface UserLookupHandler {
+        /**
+         * Performs users lookup by given [input] in suspend way. It's executed on background, so it can perform heavy operations.
+         *
+         * @param input String as user input for lookup algorithm.
+         * @return List of users as result of lookup.
+         */
+        public suspend fun handleUserLookup(input: String): List<User>
+    }
+
+    internal class DefaultUserLookupHandler(private val users: List<User>) : UserLookupHandler {
+        override suspend fun handleUserLookup(input: String): List<User> {
+            val namePattern = input.substringAfterLast("@")
```

Comment:
`input` should come here already curated.
`some mensage to mention to @oleg` ->  `oleg`

Reply:
Done